src/plugins/pdfPlayer/plugin.js

```diff
@@ -10,23 +10,47 @@ import Events from '../../utils/events.ts';
 import './style.scss';
 import '../../elements/emby-button/paper-icon-button-light';
 
+function debounce(func, wait, immediate) {
```

Comment:
You can use `debounce` from lodash module.

Reply:
Done. Thanks.